4-AdvancedGrants/2-call-api-api-ca/README.md

```diff
@@ -0,0 +1,432 @@
+1. Find the key `Enter_the_Redirect_Uri_Here` and replace the existing value with the Redirect URI for `ms-identity-javascript-tutorial-c4s1-spa`. (by default `http://localhost:3000/`).
+
+1. Open the `SPA\App\apiConfig.js` file.
+1. Find the key `Enter_the_Web_Api_Scope_Here` and replace the existing value with the scope for `ms-identity-javascript-tutorial-c4s1-middle` using the `.default` form e.g. `api://<Enter_Web_API_Application_Id>/.default`.
+
+#### Configure Known Client Applications for the middle-tier web API (ms-identity-javascript-tutorial-c4s1-middle)
+
+For a middle-tier web API (`ms-identity-javascript-tutorial-c4s1-middle`) to be able to call a downstream web API, the middle-tier app needs to be granted the required permissions as well. However, since the middle-tier cannot interact with the signed-in user, it needs to be explicitly bound to the client app in its **Azure AD** registration. This binding merges the permissions required by both the client and the middle-tier web API and presents it to the end user in a single consent dialog. The user then consent to this combined set of permissions.
+
+To achieve this, you need to add the **Application Id** of the client app, in the Manifest of the web API in the `knownClientApplications` property. Here's how:
+
+1. In the [Azure portal](https://portal.azure.com), navigate to your `ms-identity-javascript-tutorial-c4s1-middle` app registration, and select **Manifest** section.
+1. In the manifest editor, change the `"knownClientApplications": []` line so that the array contains the Client ID of the client application (`ms-identity-javascript-tutorial-c4s1-spa`) as an element of the array.
+
+    For instance:
+
+    ```json
+    "knownClientApplications": ["ca8dca8d-f828-4f08-82f5-325e1a1c6428"],
+    ```
+
+1. **Save** the changes to the manifest.
+
+### Enable Conditional Access for the downstream web API
+
+1. Inside the **Azure Active Directory** window, select the **Conditional Access** button near the bottom of the list under **Security**.
+
+![CA1](./ReadmeFiles/ca1.png)
+
+2. Select **New Policy** and name your policy.
+
+![CA3](./ReadmeFiles/ca3.png)
+
+3. Select the **Users and groups** button, select **All Users** in the **Include** tab.
+
+![CA4](./ReadmeFiles/ca4.png)
+
+4. Select the **Cloud apps**, then hit the **Select apps** radio button in the **Include** tab, and select the `ms-identity-javascript-tutorial-c4s1-downstream`.
+
+![CA5](./ReadmeFiles/ca5.png)
+
+5. Finally, select the **Grant** button and hit **Grant access**. Then check the **Require multi-factor authentication** option.
+
+![CA6](./ReadmeFiles/ca6.png)
+
+6. Enable the policy and save. Access to your web API now requires **multi-factor authentication**!
+
+## Running the sample
+
+Locate the project root folder in a command prompt. Then:
+
+```console
+    cd DownstreamAPI
+    npm start
+```
+
+```console
+    cd ..
```

Comment:
you cannot do "cd"  while npm is running :). Advise opening three command prompts